src/Tribe/Views/V2/View.php

```diff
@@ -702,6 +702,15 @@ public function get_html() {
 		 */
 		$this->repository_args = $repository_args;
 
+		/**
```

Comment:
Added a hook so that the noindex directives can be applied even when month view is cached.

Reply:
Would you mind adding the hook into the new style? `tec_events_before_view_html_cache`? We will slowly drop the V2 here.